go/admin/admin.go

```diff
@@ -77,49 +81,86 @@ func checkPassword(ctx context.Context, client *firestore.Client, password strin
 const getHTML = `
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Admin</title>
-  <style>
-    body {
-      font-family: Arial, Helvetica, sans-serif;
-      font-size: 14px;
-    }
-    .input-row {
-      display: flex;
-      padding: 5px;
-      align-items: baseline;
-    }
-    .label {
-      min-width: 100px;
-    }
-  </style>
-</head>
-<body>
-  <form enctype="multipart/form-data" method="POST">
-    <h1>Admin</h1>
-
-	<div class="input-row">
-      <span class="label">Password</span>
-      <input name="password" type="password" />
-    </div>
-
-    <h2>View scores</h2>
-    <div class="input-row">
-      <button name="action" value="scores" type="submit">View</button>
-    </div>
-
-    <h2>Update routes</h2>
-    <div class="input-row">
-      <span class="label">Areas CSV</span>
-      <input name="areas" type="file" accept=".csv" />
-    </div>
-    <div class="input-row">
-      <span class="label">Routes CSV</span>
-      <input name="routes" type="file" accept=".csv" />
-    </div>
-    <div class="input-row">
-      <button name="action" value="routes" type="submit">Upload</button>
-    </div>
-  </form>
-</body>
+  <head>
+    <title>Admin</title>
+    <style>
+      body {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 14px;
+      }
+      .input-row {
+        display: flex;
+        padding: 5px;
+        align-items: baseline;
+      }
+      .label {
+        min-width: 100px;
+      }
+    </style>
+  </head>
+  <body>
+    <form enctype="multipart/form-data" method="POST">
+      <h1>Admin</h1>
+
+      <p>A password must be supplied to perform any admin operations.</p>
+      <div class="input-row">
+        <span class="label">Password</span>
+        <input name="password" type="password" />
+      </div>
+
+      <h2>View scores</h2>
+      <p>View a scoreboard listing all teams.</p>
+      <div class="input-row">
+        <button name="action" value="scores" type="submit">View scores</button>
+      </div>
+
+      <h2>Update routes</h2>
+      <p>
+        Upload new area and route data in CSV format and use it to replace the
+        existing data.
+      </p>
+      <div class="input-row">
+        <span class="label">Areas CSV</span>
+        <input name="areas" type="file" accept=".csv" />
+      </div>
+      <div class="input-row">
+        <span class="label">Routes CSV</span>
+        <input name="routes" type="file" accept=".csv" />
+      </div>
+      <div class="input-row">
+        <button name="action" value="routes" type="submit">
+          Update routes
+        </button>
+      </div>
+
+      <h2>Delete empty teams</h2>
+      <p>Delete all teams that don't have any members.</p>
+      <div class="input-row">
+        <button name="action" value="emptyTeams" type="submit">
+          Delete empty teams
+        </button>
+      </div>
+
+      <h2>Clear scores</h2>
+      <p>
+        Clear all scores for teams and for users that aren't currently on a
+        team.
```

Comment:
I think this would be more clear if you just said "Clear scores for all teams and users".

Reply:
sure, sounds good